Modal/modal.css

```diff
@@ -0,0 +1,78 @@
+.modal-container{
+    padding:2em;
+    /* border: 2px solid black; */
+    display: flex;
+    justify-content: center;
+    
+}
+.modal{
```

Comment:
Try setting z-index for the model. It is getting overlapped by other content.
![13](https://user-images.githubusercontent.com/56469693/112980625-bb090b80-9177-11eb-97fd-1bf3c1241eff.PNG)